### PR TITLE
Weekly merge

### PR DESCRIPTION
**Pull request by silabs-oivind**

### CODE REVIEW

File: docs/user_manual/source/intro.rst [hunk 1]
```diff
@@ -99,6 +99,14 @@ In addition, the following standard instruction set extensions are available fro
      - v0.70.1 (not ratified yet; version will change)
      - always enabled
 
+   * - **Zca**: Subset of the standard **Zc** Code-Size Reduction extension consisting of a subset of **C** with the FP load/stores removed.
```
Comment (Silabs-ArjanB) [inline docs/user_manual/source/intro.rst:102]: Duplicate section; remove

File: docs/user_manual/source/preface.rst [hunk 2]
```diff
@@ -0,0 +1,15 @@
+.. _changelog:
+
```
Comment (Silabs-ArjanB) [inline docs/user_manual/source/preface.rst:2]: point to 40s

**Review by Silabs-ArjanB — CHANGES_REQUESTED** [review]

File: docs/user_manual/source/intro.rst (hunk 1 above)
Comment (silabs-oivind) [reply docs/user_manual/source/intro.rst]: Fixed

File: docs/user_manual/source/preface.rst (hunk 2 above)
Comment (silabs-oivind) [reply docs/user_manual/source/preface.rst:2]: Done

**Pull request merged**